modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
-the features of the product. You are allowed to upgrade from a release candidate to
-another release candidate and to upgrade from a previous minor version of
-{product-title} to the current release candidate. Release candidate builds are not
-supported by Red Hat and you will not be able to upgrade from a release candidate to
-the general availability release of {product-title}. Candidates should be used to
-test feature acceptance and assist in qualifying the next version of
-{product-title} in your infrastructure.
 +                                                                                    
 [NOTE]                                                                               
 ==== 
-Release candidates differ from the nightly builds found on try.openshift.com. You
+RC releases differ from the nightly builds found on try.openshift.com. You
 cannot upgrade nightly builds to nightly builds. Nightly builds are available for
 early access to features but are not upgradable or supported.
 ==== 
 
-For GA versions, the fast and stable channels present a choice between receiving
-updates as soon as they are available or allowing Red Hat to control the rollout of
-those updates.
+{product-title} {product-version} has three Upgrade Channels to choose from:
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+RC release versions are only available in candidate-{product-version}.
+
+GA release versions are only availble in fast-{product-version} or stable-{product-version}.
+
+More details about each channel and their usage are in the following sections.
+
+= candidate-{product-version}
+
+The candidate channel allows upgrading from an RC version to a later RC version and
+upgrading from a previous minor version of {product-title} to the current RC version.
+You cannot upgrade from an RC release to a GA release of {product-title}.
 
 = fast-{product-version}
 
-The fast channel is updated with new {product-version} patch versions as soon as Red
-Hat declares they are generally available. Use this channel if you wish to receive
-updates as soon as they are available or for your pre-production environments when
-participating in the connected customer program. This channel will contain all
-z-stream ({product-version}.z) updates but will not suggest upgrades to the next
-minor release (4.5.z) when the next minor release is available.
+The fast channel, containing only GA releases, is updated with new
+{product-version} patch versions as soon as Red Hat declares they are generally
+available. Use this channel if you wish to receive updates as soon as they are
+available or for your pre-production environments when participating in the connected
+customer program. This channel will contain all z-stream ({product-version}.z)
+updates but will not suggest upgrades to the next minor release (4.5.z) when the next
+minor release is available. This channel also includes the upgrade from the previous
+4.3 release.
```

Comment:
Rewording ideas; take anything that sounds useful:

> The fast-{product-version} channel is the second {product-version} channel to which releases are promoted if they have performed well in candidate-{product-version} [would be nice to link to the candidate section here].  Membership in fast-{product-version} is a declaration that the release is supported.  fast-{product-version} will never contain {product-version}.z-rc.n releases.  Some time after a release appears in fast-{product-version}, it will also appear in stable-{product-version} [would be nice to link to the stable section here].  Releases will never appear in stable-{product-version} before they appear in fast-{product-version}.

Reply:
I don't think we need to talk about previous/next minor here, because we already talked about that in the candidate section.

Reply:
We should not say `or for your pre-production environments when participating in the connected
customer program`. Because customer should take that decision. Once we have phased rollout with autoupdate, hopefully they will see the value in it and out for stable channel.

Reply:
@wking 's sugegstion `The fast-{product-version} channel is the {product-version} channel to which releases are promoted if they have performed well in candidate-{product-version} [would be nice to link to the candidate and Telemetry sections here]. Membership in fast-{product-version} is a declaration that the release is supported. ` should not be here. As by now we should have clearly mentioned that fast and stable are supported channels.